### DISCUSSION TITLE
Dependencies

### DISCUSSION THREAD

**Discussion by frankvp11** [inducer/pycuda#369] [Q&A]
What are the dependencies of Pycuda?

**Comment by inducer** [comment]
Numpy and CUDA.

**Reply by frankvp11** [reply]
Hmm, because for some reason one of my installations was trying to install matplotlib.... wierd

**Reply by inducer** [reply]
So the full truth is here:
https://github.com/inducer/pycuda/blob/8685df595b3c78e4237941c5a2c110e2020920cd/setup.py#L225-L233

But I don't see how matplotlib would become a transitive dependency of any of these.

**Reply by frankvp11** [reply]
Thats what I thought - Thanks anyways.
